codebase-stewardship/index.html

```diff
@@ -64,7 +73,9 @@ <h1>Everything your project needs to be sustainable, collaborative and open.</h1
     To drive <strong>developer success</strong> we help with (end)user support, packaging, and distributing official versions as well as developing training materials and certifications. We also help vendors such as cooperatives, NGOs and for profit companies to sell, implement and support the product.
   </p>
   </div>
-  <img alt="illustration, ecosystem" src="https://publiccodenet.github.io/illustrations/illustrations/eco-system.svg">
+
+<img alt="illustration, ecosystem" src="https://publiccodenet.github.io/illustrations/illustrations/eco-system.svg">
```

Comment:
Indentation seems correct

```suggestion
  <img alt="illustration, ecosystem" src="https://publiccodenet.github.io/illustrations/illustrations/eco-system.svg">
```